test/test_utils.py

```diff
@@ -259,3 +259,454 @@ def test_value_error_raised_if_given_invalid_formatted_date(self, date_str):
+            ),
+        ],
+    )
+    def test_table_row_string_processed_correctly_when_alignment_given_for_multiple_values(
+        self, alignment, expected
+    ):
+        # SETUP
+        values = ["Value_1", "Value_2", "Value_3"]
+        widths = [10, 11, 12]
+
+        # CALL
+        result = utils.output_table_row(values, widths, alignment=alignment)
+
+        # ASSERT
+        assert result == expected
+
+    @pytest.mark.parametrize(
+        "alignment, expected",
+        [
+            (
+                "<",
+                "Value_1   \n",
+            ),
+            (
+                "^",
+                " Value_1  \n",
+            ),
+            (
+                ">",
+                "   Value_1\n",
+            ),
+        ],
+    )
+    def test_table_row_string_processed_correctly_when_alignment_given_for_single_value(
+        self, alignment, expected
+    ):
+        # SETUP
+        values = ["Value_1"]
+        widths = [10]
+
+        # CALL
+        result = utils.output_table_row(values, widths, alignment=alignment)
+
+        # ASSERT
+        assert result == expected
+
+
+@patch("dafni_cli.utils.output_table_row")
+class TestOutputTable:
+    """Test class to test the output_table functionality"""
+
+    def test_correct_table_str_returned_for_single_column_table_with_default_alignment(
+        self, mock_output
+    ):
+        # SETUP
+        columns = ["Column_1"]
+        rows = [["Value_1"]]
+        widths = [10]
+        # setup output_table_row to return the firs column/value passed in
+        mock_output.side_effect = "Column_1  \n----------\n", "Value_1   \n"
+
+        # CALL
+        result = utils.output_table(columns, widths, rows)
+
+        # ASSERT
+        assert result == "Column_1  \n----------\nValue_1   \n"
+        assert mock_output.call_args_list == [
+            call(columns, widths, "<", header=True),
+            call(rows[0], widths, "<"),
+        ]
+
+    def test_correct_table_str_returned_for_multiple_column_and_row_table_with_default_alignment(
+        self, mock_output
+    ):
+        # SETUP
+        columns = ["Column_1", "Column_2"]
+        rows = [["Value_1", "Value_2"], ["Value_3", "Value_4"]]
+        widths = [10]
+        # setup output_table_row to return the firs column/value passed in
+        mock_output.side_effect = (
+            "Column_1  Column_2   \n----------\n",
+            "Value_1   Value_2    \n",
+            "Value_3   Value_4    \n",
+        )
+
+        # CALL
+        result = utils.output_table(columns, widths, rows)
+
+        # ASSERT
+        assert (
+            result
+            == "Column_1  Column_2   \n----------\nValue_1   Value_2    \nValue_3   Value_4    \n"
+        )
+        assert mock_output.call_args_list == [
+            call(columns, widths, "<", header=True),
+            call(rows[0], widths, "<"),
+            call(rows[1], widths, "<"),
+        ]
+
+
+class TestFileSize:
```

Comment:
Rename to `TestProcessFileSize` for consistency

Reply:
updated